bug: example1 test is flaky at the end

### Describe the bug

Sometimes the very last test

```
kubectl get deploy -n specialstuff speciald -o yaml | grep 'readyReplicas: 1'
```

fails because recent enough state has not made it back to the WDS yet. For example, here is the output from the end of a failing GitHub workflow run:

```
++ kubectl get deploy -n specialstuff speciald -o yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  annotations:
    edge.kubestellar.io/customizer: example-customizer
    kcp.io/cluster: 19cu1y6l1zlqm1lg
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"apps/v1","kind":"Deployment","metadata":{"annotations":{"edge.kubestellar.io/customizer":"example-customizer"},"name":"speciald","namespace":"specialstuff"},"spec":{"selector":{"matchLabels":{"app":"special"}},"template":{"metadata":{"labels":{"app":"special"}},"spec":{"containers":[{"env":[{"name":"EXAMPLE_VAR","value":"example value"}],"image":"library/httpd:2.4","name":"httpd","ports":[{"containerPort":80,"hostPort":8082,"name":"http","protocol":"TCP"}],"volumeMounts":[{"mountPath":"/usr/local/apache2/htdocs","name":"htdocs","readOnly":true}]}],"volumes":[{"configMap":{"name":"httpd-htdocs","optional":false},"name":"htdocs"}]}}}}
    kubestellar.io/executing-count: "1"
  creationTimestamp: "2023-10-31T21:52:09Z"
  generation: 1
  name: speciald
  namespace: specialstuff
  resourceVersion: "1803"
  uid: 61217c03-da54-42a4-bb06-5975fc13e058
spec:
  selector:
    matchLabels:
      app: special
  template:
    metadata:
      labels:
        app: special
    spec:
      containers:
      - env:
        - name: EXAMPLE_VAR
          value: example value
        image: library/httpd:2.4
        name: httpd
        ports:
        - containerPort: 80
          hostPort: 8082
          name: http
          protocol: TCP
        volumeMounts:
        - mountPath: /usr/local/apache2/htdocs
          name: htdocs
          readOnly: true
      volumes:
      - configMap:
          name: httpd-htdocs
          optional: false
        name: htdocs
status:
  conditions:
  - lastTransitionTime: "2023-10-31T21:53:47Z"
    lastUpdateTime: "2023-10-31T21:53:47Z"
    message: Deployment does not have minimum availability.
    reason: MinimumReplicasUnavailable
    status: "False"
    type: Available
  - lastTransitionTime: "2023-10-31T21:53:47Z"
    lastUpdateTime: "2023-10-31T21:53:47Z"
    message: ReplicaSet "speciald-76f6fc4cfc" is progressing.
    reason: ReplicaSetUpdated
    status: "True"
    type: Progressing
  observedGeneration: 1
  replicas: 1
  unavailableReplicas: 1
  updatedReplicas: 1
++ kubectl get deploy -n specialstuff speciald -o yaml
++ grep 'readyReplicas: 1'
make: *** [Makefile:227: docs-ecutable] Error 1
```


### Steps To Reproduce

See above

### Expected Behavior

The test has a little patience. The syncer runs periodically, the placement translator runs asynchronously.

### Additional Context

_No response_

--- a/docs/content/Coding Milestones/PoC2023q1/example1.md
+++ b/docs/content/Coding Milestones/PoC2023q1/example1.md
@@ -83,7 +83,28 @@ it has been filled in with the information from the guilder cluster.
 
 ```shell
 kubectl get deploy -n specialstuff speciald -o yaml
-kubectl get deploy -n specialstuff speciald -o yaml | grep 'readyReplicas: 1'
+```
+
+Current status might not be there yet. The following command waits for
+status that reports that there is a special workload pod "ready".
+
+```shell
+let count=1
+while true; do
+    rsyaml=$(kubectl get deploy -n specialstuff speciald -o yaml)
+    if grep 'readyReplicas: 1' <<<"$rsyaml"
+    then break
+    fi
+    echo ""
+    echo "Got:"
+    cat <<<"$rsyaml"
+    if (( count > 5 )); then
+        echo 'Giving up!' >&2
+        false
+    fi
+    sleep 15
+    let count=count+1
+done
 ```
 
 ### Status Summarization (aspirational)

